{{ .ProjectSnake }}/requirements/all.in

```diff
@@ -0,0 +1,4 @@
+# Collect all dependencies, to pass to pip.parse and create the @pip repository.
+# See README.md
```

Comment:
Nit:
```suggestion
# See ./README.md
```
(I initially looked at the README in the root of the repo, since that's the one I had open already, before I realized this was referring to the README in this subdirectory.)